bin/ofs/src/fuse.rs

```diff
@@ -636,6 +637,107 @@ impl PathFilesystem for Fuse {
             entries: relative_paths.chain(children).skip(offset as usize).boxed(),
         })
     }
+    async fn rename2(
+        &self,
+        req: Request,
+        origin_parent: &OsStr,
+        origin_name: &OsStr,
+        parent: &OsStr,
+        name: &OsStr,
+        _flags: u32,
+    ) -> Result<()> {
+        log::debug!(
+            "rename2(origin_parent={:?}, origin_name={:?}, parent={:?}, name={:?})",
+            origin_parent,
+            origin_name,
+            parent,
+            name
+        );
+        self.rename(req, origin_parent, origin_name, parent, name)
```

Comment:
We can check cap `rename` first.

Reply:
Sorry, I did not get this. Which needs to be checked?

Reply:
Check `op.info().full_capability().rename`

Reply:
checked, PTAL!